Sources/S5TF/utils/S5TFUtils.swift

```diff
@@ -127,12 +133,7 @@ public struct S5TFUtils {
         let localURL = Downloader.download(fileAt: fileAt,
                                            cacheName: cacheName,
                                            fileName: fileName)
-        do {
-            try extract(fileAt: localURL!)
-        } catch {
-            fatalError("Archive not extracted.")
-        }
-        
+        extract(fileAt: localURL!)
```

Comment:
Note that this returns the archive URL, not the URL of the extracted files. Please update the doc string.